live-build/misc/upgrade-scripts/verify

```diff
@@ -105,7 +133,7 @@ fi
 
 systemd-run --machine="$CONTAINER" --quiet --pipe --wait -- \
 	/usr/bin/java \
-	-Dlog.dir=/var/delphix/server/upgrade-verify \
+	-Dlog.dir="$UPGRADE_VERIFY_CONTAINER_LOG_DIR" \
```

Comment:
Rather than having to add the additional "copy" step to the "cleanup" function, can we instead just set:
```
-Dlog.dir="$DROPBOX_DIR/verify_logs
```
?

If so, that seems like it'd be more simple of a solution, and we wouldn't have to perform any copy operations on cleanup.

Reply:
I added the "copy" logic to easily add more log directories to copy if we needed. Users can add additional directories to `logs_dirs_to_copy` variable. Postgres log, for example, is one of them but I didn't include it to this review since I needed more info on what exact logs we need to copy.

Also, I think the "copy" logic is necessary if we do a verification logic that doesn't allow specifying log output directory but want to save those for debugging purpose. For example, `dpkg.log` maybe?

I think the logic is fairly clean as it is: there is one place to store logs and one place to copy.

Just for additional info, I introduced `$DROPBOX_DIR/verify_logs` to isolate verification logs from other irrelevant files in `$DROPBOX_DIR` directory because it's used by other services as well (`WindowsDebugService` for example), but unfortunately `support_info.sh` removes and recreates the root of `$DROPBOX_DIR` which doesn't include creating nested directories (This makes sense (kind of..) since there is no way to find out container structure of `$DROPBOX` dir), so I create it just before the copy logic such that precondition is guaranteed to satisfy. (Just for additional info, `log.dir` is used by Log4J but I don't think it creates the directory automatically if I'm not wrong).

If you still strongly feel it's not necessary, then I can make the changes.